Services/Interfaces/web_interface/models/backtesting.py

```diff
@@ -359,10 +360,11 @@ async def _start_collect_and_notify(data_collector_instance):
 
 
 async def _background_collect_exchange_historical_data(exchange, symbols, time_frames, start_timestamp, end_timestamp):
+    symbols = symbols if isinstance(symbols, list) else [symbols]
```

Comment:
How couldn't `symbols` be a list?

Reply:
I checked and I can't see any reason, I'll remove this line and see if it break things